### PR TITLE
Bridge Staff Locker

### PR DESCRIPTION
**Pull request by MayhemSailor**
## Adds Bridge Staff Locker & Maps it to Vago, Hammerhead, and Enterprise

The Locker
PR Closet

The locker keeps within the command staff color scheme, replacing the gold detailing with silver. 

PR Closet 1

The contents are as follows:
- Sunglasses
- Radio (hop)
- Jackboots
- Ship Officer Uniform
- Durathread Beret

For placement see MapDiff.

## Why It's Good For The Game

Personnel assigned as Bridge Staff mid round are left with the radio they were assigned round start and whatever clothing they've decided to wear to XO's office. This PR rectifies that situation.

## Changelog
:cl:
add: NT Uniform Systems Command has issued the Bridge Staff Uniform Locker to all in-service NSVs.
/:cl:

### CODE REVIEW

**Comment by MayhemSailor** [comment]
DNM. I am preparing closets for other NSV specific jobs.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -65,6 +65,21 @@
 	new /obj/item/storage/photo_album/HoP(src)
 	new /obj/item/card/id/departmental_budget/srv(src)
 
+//NSV Bridge Officer Locker
```
Comment (Karmic-Skink) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:68]: Move this entire thing to an NSV13 subfolder.

**Comment by Karmic-Skink** [comment]
Also please don't change the closets.dmi, please use the custom_closets.dmi in the NSV13 folder.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 1 above)
Comment (Karmic-Skink) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm:68]: Also its not a bridge officer.

**Comment by MayhemSailor** [comment]
Will be waiting for the XO PR. See you soon!

**Pull request closed without merging**